libs/sql-schema-describer/src/getters.rs

```diff
@@ -0,0 +1,52 @@
+//! Functions for fetching from quaint result rows
+
+use quaint::connector::ResultRow;
+
+pub trait Getter {
+    fn get_expect_string(&self, name: &str) -> String;
+    fn get_expect_char(&self, name: &str) -> char;
+    fn get_expect_i64(&self, name: &str) -> i64;
+    fn get_expect_bool(&self, name: &str) -> bool;
+
+    fn get_string(&self, name: &str) -> Option<String>;
+    fn get_u32(&self, name: &str) -> Option<u32>;
+    fn get_i64(&self, name: &str) -> Option<i64>;
+}
+
+impl Getter for ResultRow {
+    fn get_expect_string(&self, name: &str) -> String {
+        self.get(name)
+            .and_then(|x| x.to_string())
+            .expect(&format!("Getting {} from Resultrow {:?} as String failed", name, &self))
```

Comment:
nitpick: this will incur a string allocation for every single value, even in the success case. It can be avoided with something like 

```rust
            .and_then(|x| x.to_string())
            .ok_or_else(|| format!("Getting {} from Resultrow {:?} as String failed", name, &self)) 
            .unwrap()
```

Reply:
Adapted it.